python/veles/asrv/srv.py

```diff
@@ -49,21 +49,11 @@ def kill(self):
             self.parent.listers.remove(self)
 
     def matches(self, obj):
-        if self.pos[0] is not None:
-            if obj.pos_start is None:
-                return False
-            if obj.pos_end is not None and obj.pos_end <= self.pos[0]:
-                return False
-        if self.pos[1] is not None:
-            if obj.pos_start is not None and obj.pos_start >= self.pos[1]:
-                return False
-        for tags in self.tags:
-            for k, v in tags:
-                if v != (k in obj.tags):
-                    break
-            else:
-                return True
-        return False
+        if not self.pos.matches(obj):
+            return False
+        if not self.tags <= obj.tags:
```

Comment:
Mhm, so this fixes listers, so now we remove option to filter by objects that don't have some tag, right?